<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlFormListFilterBase</class>
 <widget class="QWidget" name="BlFormListFilterBase">
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_filtro">
        <property name="text">
         <string>Filtrar por</string>
        </property>
        <property name="buddy">
         <cstring>m_filtro</cstring>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="mui_filtro_columna">
        <property name="minimumSize">
         <size>
          <width>50</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="mui_filtro_boolean">
        <property name="maximumSize">
         <size>
          <width>20</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="checked">
         <bool>false</bool>
        </property>
        <property name="tristate">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="mui_filtro_compara">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>16777215</height>
         </size>
        </property>
        <item>
         <property name="text">
          <string>=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&gt;=</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>&lt;=</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="BlDateSearch" name="mui_filtro_fecha_ini">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BlDateSearch" name="mui_filtro_fecha_fin">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="m_filtro"/>
      </item>
      <item>
       <widget class="QComboBox" name="mui_filtro_coincidencia">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <item>
         <property name="text">
          <string>Cualquier parte del campo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Comienzo del campo</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Hacer coincidir todo el campo</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BlDateSearch</class>
   <extends>QLineEdit</extends>
   <header>bldatesearch.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">local_BlI18n.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
